readthedocs/api/v3/serializers.py

```diff
@@ -736,6 +736,28 @@ def get_admin(self, obj):
         return AdminPermission.is_admin(user, obj)
 
 
+class RestrictedProjectSerializer(serializers.ModelSerializer):
```

Comment:
I'd prefer if we name this `UnAuthedProjectSerializer` or `AnonymousProjectSerializer` which is more explicit _when_ this serializer is going to be used.